src/main/java/com/bio4j/model/uniprot/programs/XMLConstants.java

```diff
@@ -81,6 +81,29 @@
   public static final String BOOK_CITATION_TYPE                     = "book";
   public static final String UNPUBLISHED_OBSERVATION_CITATION_TYPE  = "unpublished observations";
 
+  // this is a static representation of the entry XML structure
+  interface ENTRY {
+    String element = "entry";
+    interface ACCESSION {
+      String element = "accession";
+    }
+    interface COMMENT {
+      String element = "comment";
+      interface TYPE {
+        String attribute = "type";
+        public static final String INTERACTION = "interaction";
+      }
+      interface INTERACTANT     { String element = "interactant";
+        interface INTACTID  { String attribute  = "intactId"; }
+        interface ID        { String element    = "id";       }
+      }
+      interface ORGANISMSDIFFER { String element    = "organismsDiffer";  }
+      interface EXPERIMENTS     { String element    = "experiments";      }
+    }
+  }
```

Comment:
what? :sweat_smile: ok, not objects, some `very final protected public class ...`?